src/monodroid/CMakeLists.txt

```diff
@@ -60,7 +60,7 @@ if(NOT ANDROID)
   endforeach()
   set(MONODROID_SOURCES
     ${MONODROID_SOURCES}
-    ${SOURCES_DIR}/inmemory-assemblies.cc
+    ${SOURCES_DIR}/sideloaded-assemblies.cc
```

Comment:
Should we just call it `designer-assemblies.cc` and be done with it? :-)

Reply:
Up to you, if you think it makes it clearer what the purpose is

Reply:
@jonpryor do you still want that rename?

Reply:
renamed